pkg/store/flushable.go

```diff
@@ -20,23 +19,10 @@ type flushableServer interface {
 
 func newFlushableServer(
 	upstream storepb.Store_SeriesServer,
-	labelNames stringset.Set,
-	replicaLabels []string,
 ) flushableServer {
-	if labelNames.HasAny(replicaLabels) {
-		return &resortingServer{Store_SeriesServer: upstream}
-	}
-	return &passthroughServer{Store_SeriesServer: upstream}
+	return &resortingServer{Store_SeriesServer: upstream}
```

Comment:
Should we put this change under a feature flag? I think this doesn't make sense for Cortex usecase and introduces additional performance overhead.
It is still good to keep the `passthroughServer`

Reply:
Of course! Out of curiosity how does cortex deal with this issue?

Reply:
For cortex it is always passthrough... It is like no replica labels in block (only 1 reserved label)

Reply:
ill make it configurable in a followup PR